Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -4715,6 +4715,74 @@ def panorama_add_static_route_command():
     })
 
 
+def panorama_override_vulnerability(threatid: str, vulnerability_profile: str, drop_mode: str):
+    xpath = "{}profiles/vulnerability/entry[@name='{}']/threat-exception/entry[@name='{}']/action".format(
+        XPATH_OBJECTS,
+        vulnerability_profile,
+        threatid)
+    params = {'action': 'set',
+              'type': 'config',
+              'xpath': xpath,
+              'key': API_KEY,
+              'element': "<{0}></{0}>".format(drop_mode)
+              }
+
+    return http_request(
+        URL,
+        'POST',
+        body=params,
+    )
+
+
+@logger
+def panorama_get_predefined_threats_list(target: str):
+    """
+    Get the entire list of predefined threats as a file in Demisto
+    """
+    params = {
+        'type': 'op',
+        'cmd': '<show><predefined><xpath>/predefined/threats</xpath></predefined></show>',
+        'target': target,
+        'key': API_KEY
+    }
+    result = http_request(
+        URL,
+        'GET',
+        params=params
+    )
+    demisto.results(result['response']['result'])
```

Comment:
we use the interior func to return it's result to the `command`, and do the `demisto.results()` from there.